cdap-master/src/test/java/io/cdap/cdap/master/environment/k8s/MasterServiceMainTestBase.java

```diff
@@ -109,15 +107,18 @@ public static void init() throws Exception {
                    RemoteProgramRunRecordFetcher.class, ProgramRunRecordFetcher.class);
 
     // Starting all master service mains
-    List<Class<? extends AbstractServiceMain<EnvironmentOptions>>> serviceMainClasses =
+    List<Class<? extends AbstractServiceMain<EnvironmentOptions>>> serviceMainClasses = new ArrayList<>(
       Arrays.asList(RouterServiceMain.class,
                     MessagingServiceMain.class,
                     MetricsServiceMain.class,
                     LogsServiceMain.class,
                     MetadataServiceMain.class,
                     RuntimeServiceMain.class,
-                    AppFabricServiceMain.class,
-                    AuthenticationServiceMain.class);
+                    AppFabricServiceMain.class));
+
+    if (cConf.getBoolean(Constants.Security.ENABLED)) {
```

Comment:
Please add a check to make sure we are using MANAGED mode here for [Constants.Security.Authentication.AUTHENTICATION_MODE](https://github.com/cdapio/cdap/blob/develop/cdap-common/src/main/java/io/cdap/cdap/common/conf/Constants.java#L1066), thanks!

Reply:
fixed